.gitignore

```diff
@@ -1,4 +1,3 @@
 .idea
 node_modules
 coverage
-package-lock.json
```

Comment:
This is something we should not publish within the library code. Users might lock packages versions on their project level.